charts/external-dns/templates/servicemonitor.yaml

```
@@ -1,9 +1,13 @@
-{{- if.Values.serviceMonitor.enabled -}}
+{{- if .Values.serviceMonitor.enabled -}}
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ include "external-dns.fullname" . }}
-  namespace: {{ .Release.Namespace }}
+  namespace: {{ .Values.serviceMonitor.namespace | default .Release.Namespace }}
```

```suggestion
  namespace: {{ default .Release.Namespace .Values.serviceMonitor.namespace }}
```

I think this would be clearer.